source/kraken/apply_disruption.cpp

```diff
@@ -716,8 +722,7 @@ struct add_impacts_visitor : public apply_impacts_visitor {
             mvj->push_unique_impact(impact);
 
             auto nb_rt_vj = mvj->get_vjs_at(rt_level).size();
-            std::string new_vj_uri = "vehicle_journey:" + mvj->uri + ":" + type::get_string_from_rt_level(rt_level)
-                                     + ":" + std::to_string(nb_rt_vj) + concatenate_impact_uris(*mvj);
```

Comment:
This `concatenate_impact_uris()` should be removed, it's unused after that.

I guess the uri leak was on that one, right?